src/components/protected-areas-table/component.jsx

```diff
@@ -187,7 +187,17 @@ function ProtectedAreasTable({ data, handleSortChange }) {
           data.map((row, index) => (
             // eslint-disable-next-line react/no-array-index-key
             <tr key={`wdpa-row-${row.NAME}-${index}`}>
-              <td className={styles.firstColumn}>{row.NAME}</td>
+              <td className={styles.firstColumn}>
+                <a
+                  href={
```

Comment:
This should open on an external page. Please include a title attribute too for accessibility